frontend/dist/js/common.js

```diff
@@ -0,0 +1,29 @@
+var Common = function () {
+    var common = {};
+    //Fetch user info from server and return json object
+    common.fetchInfo = function (social) {
+        $.ajax({
+                type: 'GET',
+                url: 'http://quadcore.news/api/login/' + social,
+            })
+            .done(function (data) {
+                if (data.result == 0) {
+                    return data;
+                } else {
+                    alert("Fecthing info failed");
```

Comment:
Do we want to show this message to the user? A word like 'fail' gives users a very very bad experience, so maybe we can show some other message, or think about other ways (e.g. A message in the UI rather than an alert popup, or just showing nothing)

Reply:
It's for just checking

Reply:
:)